flowdb/sql/04_schema_other.sql

```diff
@@ -80,3 +80,23 @@ CREATE TABLE etl.etl (
 	time_stamp TIMESTAMP WITH TIME ZONE,
 	PRIMARY KEY (id)
 );
+
+/*
+
+Schema used for record keeping of aggregate calculations.
+
+*/
+
+CREATE SCHEMA IF NOT EXISTS aggregates;
```

Comment:
This name suggests to me that the schema will contain _results_ of aggregated query calculations, rather than metadata on the calculations. I'd prefer a slightly more expressive name (maybe `calculations_metadata` or `record_keeping` or something similar?). Any particular opinions @greenape?

Reply:
It's going to be completely changed - just need it ASAP

Reply:
It's a terrible name...

Reply:
Sure, I understand. It's just that temporary fixes have a tendency to stick around for longer than one would like, so good to choose an explicit terrible name that helps others understand at a glance what it's for. ;)